src/Template/Adrs/xml/e2b.ctp

```diff
@@ -220,7 +224,7 @@
                     elseif(strtolower($adrListOfDrug['relationship_to_sae']) == 'pending') { echo 2; }
                     else { echo 2; }
                 ?></drugcharacterization>
-                <medicinalproduct><?php echo $adrListOfDrug['drug_name']; ?></medicinalproduct>
+                <medicinalproduct><?php echo preg_replace('/[^A-Za-z0-9\s+]/', '',$adrListOfDrug['drug_name']); ?></medicinalproduct>
```

Comment:
Create a utility method that will perform the replace function and reuse it as appropriate